usage/jsgui/src/main/license/files/LICENSE

```diff
@@ -353,6 +353,13 @@ This project includes the software: ZeroClipboard
   Used under the following license: The MIT License (http://opensource.org/licenses/MIT)
   Copyright (c) Jon Rohan, James M. Greene (2014)
 
+This project includes the software: CodeMirror
```

Comment:
looks like you manually added this?  this file is autogenerated based on `overrides.yaml` which is where this data should be added.  (and it will update other `LICENSE` files as well because we are obligated to include info this not just in the UI license file but in the source and binary dists as well.)